assets/csidriver.yaml

```diff
@@ -2,6 +2,9 @@ kind: CSIDriver
 apiVersion: storage.k8s.io/v1
 metadata:
   name: efs.csi.aws.com
+  annotations:
+    # This CSIDriver is managed by an OCP CSI operator
+    csi.openshift.io/managed: "true"
```

Comment:
I know this is a unrelated change, but we were missing this annotation and I had this as uncommitted change sitting in my working tree for some reason. I can open a different PR, if need be.